readme.md

```diff
@@ -46,6 +46,23 @@ $ curl -sSLO https://github.com/sindresorhus/run-node/raw/master/run-node && chm
 ```
 ./run-node file.js
 ```
+### Add and edit customizable cache path and error message
+
+The cache path and error message are defined by ```RUN_NODE_CACHE_PATH``` and ```RUN_NODE_ERROR_MSG``` respectively. To add and edit this variables add this line to ```~.bashrc``` or ```~/.bash_profile```:
```

Comment:
You can use single backticks instead of 3.

Reply:
And this is not just meant for your bashrc, but also for scripts that want to use set a different cache path or message. This should be mention too.

Reply:
Your readme used 3 backticks instade of 1 so I kept like it was...

Reply:
I will mention that it is also meant for scripts

Reply:
You should use single backticks for inline code (e.g. in a sentence) and triple backticks for code blocks, see https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting.

Reply:
Ah, I see, sorry I did not know that... I will update this in the next hour.